src/pykiso/pytest_plugin/collection.py

```diff
@@ -0,0 +1,331 @@
+##########################################################################
+# Copyright (c) 2010-2023 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+Test collection
+***************
+
+:module: collection
+
+:synopsis: allow pytest to collect test suites based on pykiso YAML
+    configuration files. Make defined auxiliaries available as fixtures.
+
+"""
+
+from __future__ import annotations
+
+from collections import defaultdict
+from functools import partial
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import pytest
+from _pytest.fixtures import FixtureDef
+from _pytest.unittest import TestCaseFunction
+
+from pykiso.config_parser import parse_config
+from pykiso.interfaces.dt_auxiliary import DTAuxiliaryInterface
+from pykiso.test_coordinator.test_execution import apply_tag_filter
+from pykiso.test_coordinator.test_suite import tc_sort_key
+from pykiso.test_setup.config_registry import ConfigRegistry
+
+from .utils import *
```

Comment:
from .utils import export? Or is there any reason to use a wild import for utils?

Reply:
This imports export, get_base_testcase and is_kiso_testcase (all exported functions)
So yes there's a reason ;)